.github/workflows/release-nightly.yml

```diff
@@ -49,6 +49,7 @@ jobs:
     needs: build_wheels
     runs-on: ubuntu-24.04
     steps:
+      - uses: actions/checkout@v4
```

Comment:
Does this not need https://github.com/scipp/scipp/blob/7dd14804a37dffa100d09edadec20ac689c66393/.github/workflows/release.yml#L52 ?

Reply:
This job doesn't build anything, just uploads the wheel already built.